### PR TITLE
should be good

### PR DESCRIPTION
**Pull request by doutriaux1**

### CODE REVIEW

**Comment by doutriaux1** [comment]
@ocefpaf I'm not sure why travis kept building for py3k until I forced python<3  but it's all green and ready for review now. Thanks.

File: recipes/genutil/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+source:
+    git_rev: {{ version }}
+    git_url: git://github.com/UV-CDAT/genutil.git
+
+build:
+  number: 0
+
+requirements:
+  build:
+    - python <3
```
Comment (ocefpaf) [inline recipes/genutil/meta.yaml]: That is not the way to skip Python 3 build. See my other comment.

File: recipes/genutil/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+package:
+    name: {{ name|lower }}
+    version: {{ version }}
+
+source:
+    git_rev: {{ version }}
+    git_url: git://github.com/UV-CDAT/genutil.git
+
+build:
+  number: 0
```
Comment (ocefpaf) [inline recipes/genutil/meta.yaml]: Please add a skip statement like,

```yaml
skip: True  # [not py3k]
```

in the `build` section.

File: recipes/genutil/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+
+requirements:
+  build:
+    - python <3
+    - setuptools
+    - numpy x.x
+    - udunits2
+    - toolchain  # [osx]
+  run:
+    - python <3
```
Comment (ocefpaf) [inline recipes/genutil/meta.yaml]: Same as before, remove the `<3` and let the skip statement take care of it.

**Comment by doutriaux1** [comment]
@ocefpaf I think we can merge now. Correct?

File: recipes/genutil/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,41 @@
+{% set name = "genutil" %}
+{% set version = "2.8.2" %}
+
+package:
+    name: {{ name|lower }}
+    version: {{ version }}
+
+source:
+    git_rev: {{ version }}
+    git_url: git://github.com/UV-CDAT/genutil.git
```
Comment (ocefpaf) [inline recipes/genutil/meta.yaml]: Can you change to the tarball format?

**Comment by doutriaux1** [comment]
I'd rather not because we have a lot of packages and when tagging a new release, it's much simplier to simply edit the tag point in the recipe thaan going through each package download the tarball compute the sha1  etc... If you prefer I can put an actual sha1 rather than a tag point for the git repo.

**Comment by ocefpaf** [comment]
> I'd rather not because we have a lot of packages and when tagging a new release, it's much simplier to simply edit the tag point in the recipe thaan going through each package download the tarball compute the sha1 etc... If you prefer I can put an actual sha1 rather than a tag point for the git repo.

I understand that is more convenient for you but in `conda-forge` we chose the tarball format not only because of the the checksum but to reduce the download time and reduce the strain on the CIs.

**Comment by doutriaux1** [comment]
ahh... I see! I didn't realize that. Will change this then.

**Comment by ocefpaf** [comment]
> I see! I didn't realize that.

I didn't explain it first :grimacing: 

> Will change this then.

Thanks!

**Comment by doutriaux1** [comment]
@ocefpaf once this is in how long should I wait before I can create a PR that depends on this?

File: recipes/genutil/meta.yaml [hunk 5]
```diff
@@ -0,0 +1,42 @@
+{% set name = "genutil" %}
+{% set version = "2.8.2" %}
+
+package:
+    name: {{ name|lower }}
+    version: {{ version }}
+
+source:
+    url: https://github.com/UV-CDAT/genutil/archive/{{ version }}.tar.gz
```
Comment (ocefpaf) [inline recipes/genutil/meta.yaml]: I guess that the `fn:` field is no longer mandatory. I just learned a new thing :smile:

**Comment by ocefpaf** [comment]
> @ocefpaf once this is in how long should I wait before I can create a recipe that depends on this one?

As soon as you receive the feedstock e-mail you can follow the CI builds there. But that may take a while b/c staged-recipes is under some strain this week.

**Pull request merged**